cases/signals.py

```diff
@@ -1,8 +1,38 @@
 from django.db.models.signals import post_save
-from django.dispatch import receiver
+from django.dispatch import receiver, Signal
+
+from accounts.models import User
+from noiseworks import cobrand
+from noiseworks.message import send_email
 
 from .models import Action, Case, Complaint, MergeRecord
 
+new_case_reported = Signal()
+
+
+@receiver(new_case_reported)
+def auto_assign_new_case(sender, case, case_absolute_url, **kwargs):
+    if not case.assigned and case.estate != "y" and case.ward:
```

Comment:
This is fine, I generally feel in these type of functions it's a bit clearer if you reverse the if statements and then return out of the function, only to prevent the triple indented code further down, but no biggie on a small function.